### PR TITLE
Create 2021-09-01---emergency-release.md

### PR DESCRIPTION
**Pull request by umahasit**

### CODE REVIEW

**Comment by umahasit** [comment]
..TBC id to be updated upon release

File: products/waf/src/content/change-log/2021-09-01---emergency-release.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+type: table
+order: 930
+---
+
+# 2021-09-01
```
Comment (pedrosousa) [inline products/waf/src/content/change-log/2021-09-01---emergency-release.md]:
```suggestion
# 2021-09-01 – Emergency Release
```

**Review by pedrosousa — APPROVED** [review]

**Comment by pedrosousa** [comment]
Sorry, there's also a `title: ` missing in the Markdown frontmatter:

```
title: "2021-09-01 – Emergency"
```

**Comment by pedrosousa** [comment]
LGTM

**Pull request merged**